zwad/ad/__init__.py

```diff
@@ -13,7 +13,7 @@
 from zwad.ad.util import run_classifier, fetch_anomalies
 
 
-class ZtfAnomalyDetector:
+class BaseAnomalyDetector:
```

Comment:
What about make it ABC?

```python
from abc import ANC, abstractmethod

class BaseAnomalyDetector(ABC):
    ...
    @abstractmethod
    def _load_dataset()
```

Reply:
ABC normally stands for Approximate Bayesian Computation... I would refrain from using such nomenclature with a different meaning..

PS: I have no idea if this comment makes sense :)

Reply:
It is not me, it is Guido
https://docs.python.org/3/library/abc.html

Reply:
as expected, my comment makes no sense.. please ignore it :)

I am just happy to be part of the conversation ...

Reply:
> What about make it ABC?
> 
> ```python
> from abc import ANC, abstractmethod
> 
> class BaseAnomalyDetector(ABC):
>     ...
>     @abstractmethod
>     def _load_dataset()
> ```

Thanks. Done